module/Decision/src/Decision/Service/Decision.php

```diff
@@ -257,6 +264,58 @@ public function deleteDocument($post)
         $document = $this->getMeetingDocument($id);
         $this->getMeetingMapper()->remove($document);
     }
+
+    /**
+     * Changes a document's position in the ordering
+     *
+     * @param int $id Document ID
+     * @param bool $moveDown If the document should be moved down in the ordering, defaults to TRUE
+     * @return void
+     * @throws \InvalidArgumentException If the document doesn't exist
+     */
+    public function changePositionDocument($id, $moveDown = true)
```

Comment:
The basic flow is (1) retrieve documents, (2) swap document positions, then (3) persit position. Unfortuantly, I have to update the positions of all documents related to a meeting because of legacy. Old documents don't have a positon yet, so they are set to 0 by default. In the future, when documents have display positions, we can simplify this by only mutating two rows.